boards/nucleo-l432kc/Makefile.features

```diff
@@ -2,6 +2,7 @@
 FEATURES_PROVIDED += periph_pwm
 FEATURES_PROVIDED += periph_rtc
 FEATURES_PROVIDED += periph_rtt
+FEATURES_PROVIDED += periph_i2c
```

Comment:
Please keep the alphabetical order as described in the comment above :)

Reply:
✅